src/test/enhetstester/kotlin/no/nav/familie/ba/sak/kjerne/behandling/domene/BehandlingSøknadsinfoServiceTest.kt

```diff
@@ -0,0 +1,77 @@
+package no.nav.familie.ba.sak.kjerne.behandling.domene
+
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import no.nav.familie.ba.sak.datagenerator.lagBehandling
+import no.nav.familie.ba.sak.kjerne.behandling.Søknadsinfo
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.time.LocalDate
+
+class BehandlingSøknadsinfoServiceTest {
+    private val behandlingSøknadsinfoRepository = mockk<BehandlingSøknadsinfoRepository>(relaxed = true)
+    private val behandlingSøknadsinfoService = BehandlingSøknadsinfoService(behandlingSøknadsinfoRepository)
+    private val behandling = lagBehandling()
+    private val journalpostId = "123456789"
+    private val søknadMottattDato = LocalDate.of(2025, 1, 1)
+    private val søknadsinfo =
+        Søknadsinfo(
+            journalpostId = journalpostId,
+            brevkode = "BREV123",
+            erDigital = true,
+        )
+
+    @BeforeEach
+    fun setUp() {
+        every { behandlingSøknadsinfoRepository.save(any()) } returns mockk()
```

Comment:
Har kanskje ikke så mye å si men gir det mer mening å bare returnere argumentet som sendes inn`

```suggestion
        every { behandlingSøknadsinfoRepository.save(any()) } returnsArgument 0
```